applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -118,17 +119,17 @@ static void stream_started_cb(struct bt_audio_stream *stream)
 	LOG_INF("Broadcast source %p started", (void *)stream);
 }
 
-static void stream_stopped_cb(struct bt_audio_stream *stream)
+static void stream_stopped_cb(struct bt_bap_stream *stream, uint8_t reason)
 {
 	int ret;
 
 	ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_NOT_STREAMING);
 	ERR_CHK(ret);
 
-	LOG_INF("Broadcast source %p stopped", (void *)stream);
+	LOG_INF("Broadcast source %p stopped. Reason: %d", (void *)stream, reason);
```

Comment:
The reason should be DBG

Reply:
As discussed on Teams, this will be addressed when we do the architecture rewrite